ISHITA/calc.css

```diff
@@ -0,0 +1,41 @@
+.container
+{
+    text-align: center;
+    margin-top: 30px;
+}
+table
+{
+    margin: auto;
+}
+input
+{
+    font-size: medium;
+    width: 200px;
+    height: 80px;
+    border: 2px solid rgb(148, 141, 141);
+    border-radius: 9px;
+}
+
+button
+{
+    font-size: large;
+    background: rgb(3, 3, 3);
+    width: 50px;
+    height: 50px;
+    border-radius: 4px;
+    color:white;
+}
+.calculator
+{
+    display: inline-block;
+    margin: 8px;
+    background-color: wheat;
+    padding: 10px;
+    border-radius: 8px;
+    height: 48vh;
```

Comment:
I readjusted the vertical height . 
```suggestion
    height: 58vh;
```
![Screenshot (194)](https://user-images.githubusercontent.com/60812924/125198741-ce506a80-e280-11eb-8277-5fb41f442529.png)